ratings/ratings.py

```diff
@@ -0,0 +1,298 @@
+"""
+Copyright 2018 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+""" 
+
+import sys
+sys.path.insert(0, '.')
+
+from absl import flags
+
+import choix
+import numpy as np
+import sqlite3
+import os, os.path
+import re
+import fsdb
+import random
+import math
+from tqdm import tqdm
+import datetime as dt
+
+
+EVAL_REGEX = "(\d*)-minigo-cc-evaluator-"
+MODEL_REGEX = "(\d*)-(.*)"
```

Comment:
probably you want (\d*)-([a-z-]*[a-z]) but that can still eat extra trailing words